include/zephyr/bluetooth/services/ias.h

```diff
@@ -17,10 +17,29 @@
  * as a part of ongoing development.
  */
 
+#include <zephyr/bluetooth/conn.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+enum bt_ias_alert_lvl {
+	/**
+	 * @brief No alerting should be done on device
+	 */
```

Comment:
```suggestion
	/** @brief No alerting should be done on device */
```

`@brief` can also be omitted when it's a single line:
```suggestion
	/** No alerting should be done on device */
```

Ditto for the following enum values